contracts/container/contract.go

```diff
@@ -582,6 +582,48 @@ func validatePlacementIndex(ctx storage.Context, cID interop.Hash256, inx uint8)
 	}
 }
 
+// VerifyPlacementSignatures verifies that message has been signed by container
+// members according to container's placement policy: there should be at least
+// REP number of signatures for every placement vector. sigs must be container's
+// number of SELECTs length.
+func VerifyPlacementSignatures(cid interop.Hash256, msg []byte, sigs [][]interop.Signature, curveHash crypto.NamedCurveHash) bool {
```

Comment:
Let's drop `curveHash` and simplify interface here. We're 256r1 only for this.

Reply:
I also had some thought on special `msg` for verification scripts, but maybe we can leave it as is. `msg` can be created like here in the verification script: https://github.com/nspcc-dev/neo-go/blob/a9242535db757faba860e21c194033f30c679f48/pkg/core/native/native_test/cryptolib_verification_test.go#L173 and then only `sigs` are a part of invocation script (as they should be).